integrationExamples/gpt/fledge_example.html

```diff
@@ -3,7 +3,7 @@
   <!--
     FLEDGE (Protected Audience API) configuration with GPT and FLEDGE-supporting adapter
 
-    gulp serve --modules=fledgeForGpt,openxBidAdapter
+    gulp serve --modules=paapiForGpt,openxBidAdapter
```

Comment:
in general shall we set auction config before targeting? We might want to prepare for the idea it will be set before the auction is complete.

Reply:
perhaps also rename this file